.github/workflows/integration_test_calamari.yml

```diff
@@ -262,8 +262,9 @@ jobs:
           cd ${{ github.workspace }}/Manta/tests
           yarn install
           yarn
-          yarn correctness_test --address=ws://127.0.0.1:9921 --exit
-          yarn performance_test --address=ws://127.0.0.1:9921 --exit
```

Comment:
So, what happens to these tests ?

Reply:
I think this two test is on demo-level and dummy and useless.
Just remove those is ok.

Reply:
So how is correctness/performance for the old api tested now ?

Reply:
`correctness_test` is replaced by `validate-pull-ledger-diff`, but for `performance_test`, yeah, you're right, we should keep it. I will bring it back.

Reply:
`performance_test` is back, and it supports `dense_pull_ledger_diff` right now.